app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,43 @@
+class TagsController < ApplicationController
+  before_action :authenticate_user!
+  before_action -> { check_permission('admin') }
+  before_action :set_tag, only: [:edit, :update, :destroy]
+
+  def edit
```

Comment:
now that this exists, i thought it might be nice to add a link to the tag edit page from the tag show page as part of #234 . thoughts?

Reply:
sure (so long as only admins can see the route)